packages/cli/src/cmds/validator/options.ts

```diff
@@ -61,6 +62,12 @@ export const validatorOptions: ICliCommandOptions<IValidatorCliArgs> = {
     type: "string",
   },
 
+  dontSubmitAttestationEarly: {
```

Comment:
Can we hidden:true this?

Reply:
I changed the option to `earlyAttestationDelayMs` and use `hidden:true` for it